views/matrixsplit.html

```diff
@@ -233,8 +233,8 @@
             =
             \begin{bmatrix}
```

Comment:
I'm not sure I understand this comment. Each change I did was only relying upon sign manipulation that is valid in general, namely:

    -(z-1) == (1-z)
    (z-1)^2 == (1-z)^2

These apply for whatever value of z.